substrate/frame/revive/src/evm/api/rpc_types.rs

```diff
@@ -17,8 +17,18 @@
 //! Utility impl for the RPC types.
 use super::*;
 use alloc::vec::Vec;
+use codec::{Decode, Encode};
+use frame_support::DefaultNoBound;
+use scale_info::TypeInfo;
 use sp_core::{H160, U256};
 
+/// Configuration specific to a dry-run execution.
+#[derive(Debug, Encode, Decode, TypeInfo, Clone, DefaultNoBound)]
+pub struct DryRunConfig<Moment> {
+	/// Optional timestamp override for dry-run in pending block.
+	pub timestamp_override: Option<Moment>,
```

Comment:
Otherwise clients sending an old (smaller) version of the structure will fail. `None` will aways have the same encoding no matter the type argument.

```suggestion
	/// Optional timestamp override for dry-run in pending block.
	pub timestamp_override: Option<Moment>,
	/// Used for future extensions without breaking encoding.
	pub reserved: Option<()>,
```